README.md

```diff
@@ -1,2 +1,61 @@
-# comp110-worksheet-5
+### comp110-worksheet-5
 Base repository for COMP110 worksheet 5
+
+
+### Question A
+
+The algorithm compares the two lists; it compares all the items and sees if they match. If it finds any matches then it returns "True", if not then it returns "False" and exits the for loop.
```

Comment:
Actually there is only one list -- the algorithm checks it for duplicate elements